rln/www/index.js

```diff
@@ -0,0 +1,108 @@
+import * as rln from "rln";
+import * as wc from "./witness_calculator"
+import * as files from './files'
+
+rln.init_panic_hook();
+
+function _base64ToArrayBuffer(base64) {
```

Comment:
This helper function is for converting the circom, zkey and vk files from base64 to an `Uint8Array`